yjit/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+// Silence dead code warnings until we are done porting YJIT
```

Comment:
Is this still necessary?

Reply:
Yeah, still needed for now. We've accumulated a bunch of these and will need to do a pass to address them.